docs/using-the-compiler.rst

```diff
@@ -323,6 +323,9 @@ Input Description
         },
         // Metadata settings (optional)
         "metadata": {
+          // The CBOR metadata is appended at the end of the bytecode by default.
+          // Setting this to false omits the metadata from the deployed code.
```

Comment:
Still not sure why this is restricted to "deployed code", when in fact it isn't.

Reply:
What else?

Reply:
The original suggestion by you was "runtime and deploy time code." Wouldn't deployed code capture both?

Reply:
initcode and deployed code both contain it.

Reply:
Fixed this comment.